backend/tests/v5/helper/services.js

```diff
@@ -566,8 +567,17 @@ ServiceHelper.generateGroup = (isSmart = false, {
 	if (isSmart) {
 		group.rules = [
 			{
+				name: ServiceHelper.generateRandomString(),
 				field: 'IFC GUID',
-				operator: 'IS',
+				operator: OPERATORS.IS.name,
+				values: [
+					'1rbbJcnUDEEA_ArpSqk3B7',
```

Comment:
you didn't crate it but we should probably just do generateRandomString() here (and the one below)